src/sdk/modules/passkeyValidator/toWebAuthnKey.ts

```diff
@@ -0,0 +1,176 @@
+import { startAuthentication, startRegistration } from "@simplewebauthn/browser"
```

Comment:
🤔

Reply:
Are you sure this should be adding to our environment-agnostic sdk? It looks very tied to a browser. Is there nothing in this package that uses browser services?

Reply:
Yes I'm sure, this an SDK to client side feature and the @simplewebauthn/browser exports the required functions to manage authentication and registration.